exe_12.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+
+def removing_single_dimensional_entries(array:np)->np:

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html